src/ProgressOnderwijsUtils/Html/HtmlHelpers.cs

```diff
@@ -2,17 +2,6 @@ namespace ProgressOnderwijsUtils.Html;
 
 public static class HtmlHelpers
 {
-    public static THtmlTag _class<THtmlTag>(this THtmlTag inlineElement, params string?[]? classes)
```

Comment:
Heb je gechecked of deze nog in pnet gebruikt werdt?  Zo ja is het mogelijk logischer om deze later pas te verwijderen.

Reply:
ik laat hem eerst wel staan voor het geval dat